src/deprecated.jl

```diff
@@ -1,28 +1,10 @@
 # This file contains code that was formerly a part of Julia. License is MIT: http://julialang.org/license
 
 using Base: @deprecate, depwarn
-using Compat: @dep_vectorize_1arg, @dep_vectorize_2arg
-
-for f in (:digamma, :trigamma, :zeta, :eta, :erfcx, :erfi, :dawson, :airyai, :airyaiprime,
-          :airybi, :airybiprime, :besselj0, :besselj1, :bessely0, :bessely1, :erf, :erfc)
-    @eval @dep_vectorize_1arg Number $f
-end
-
-for f in (:invdigamma, :erfinc, :erfcinv)
-    @eval @dep_vectorize_1arg Real $f
-end
-
-for f in (:polygamma, :zeta, :besseli, :besselix, :besselj, :besseljx, :besselk, :besselkx,
-          :bessely, :besselyx, :besselh, :besselhx, :hankelh1, :hankelh2, :hankelh1x, :hankelh2x)
-    @eval @dep_vectorize_2arg Number $f
-end
 
 @deprecate airy(z::Number) airyai(z)
```

Comment:
presumably these were introduced in the last release, and hence require on more version bump before removal?